.gitignore

```diff
@@ -18,3 +18,5 @@ data/*.traineddata
 wackenroder_herzensergiessungen_*.gt.txt
 wackenroder_herzensergiessungen_*.tif
 master.zip
+.vscode
```

Comment:
Both additions are not necessary by the project, but only useful for some users. Such entries can be made in your local `.git/info/exclude` or in your global Git configuration. I prefer that. Otherwise .gitignore typically grows with more and more unrelated entries.

Reply:
I didn't realize this before. Thanks for the hint, I will follow your suggestion!